raiden-ts/src/transport/epics/webrtc.ts

```diff
@@ -107,37 +107,39 @@ async function getMatrixIceServers(matrix: MatrixClient): Promise<RTCIceServer[]
   return servers;
 }
 
-// creates a filter function which filters valid MatrixEvents
-function filterMatrixVoipEvents<T extends 'offer' | 'answer' | 'candidates' | 'hangup'>(
+type ExtMatrixEvent = MatrixEvent & {
```

Comment:
Nice that you moved that out (had a comment on this the commit before). But why must the `codecTypes` remain in the functions body? :thinking:

Reply:
Right.. forgot to takeout `codecTypes`, I'll do, thanks.